src/components/measurements/measurementsD3.js

```diff
@@ -264,6 +266,7 @@ export const drawMeasurementsSVG = (ref, xAxisRef, svgData) => {
         // but there're always limits of the available space so punting that for now.
         //    -Jover, 20 September 2022
         g.selectAll('text')
+          .attr("class", classes.groupingValue)
```

Comment:
> I wish there was a static visual cue to the user that they can click on a row of the measurements panel to color by those measurements and maybe an on-hover description of what would happen on click.

I can add an [SVG title](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) here to add a basic tooltip to highlight the function.

Reply:
> When I group measurements by subclade and color the tree by those measurements, I don’t have an indication of how the resulting color values are calculated. I would assume that the values are the mean or median for each test virus across all references in the measurements group. What if we reported that information somehow when coloring by measurements across more than one reference?

Reporting when coloring by more than one reference seems like a very flu titer data specific request. Maybe this can be generalized in the on-hover description? 

```
Click to color by an average of all measurement values for a test strain in this group
```

Reply:
Added tooltip in 9f6b03260d2df285542ee5faa611dbca92353871